<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>655</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Island Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/icons/island32.png</normaloff>:/images/icons/island32.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>622</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabIslandManagement">
       <attribute name="title">
        <string>Manage</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="resources.qrc">:/images/icons/island256.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
              <property name="spacing">
               <number>9</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SetMaximumSize</enum>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>24</number>
              </property>
              <item>
               <widget class="QLabel" name="islandStatusLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>1965</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Island status: </string>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="islandStatus">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>1965</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="palette">
                 <palette>
                  <active>
                   <colorrole role="WindowText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>173</red>
                      <green>173</green>
                      <blue>173</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </active>
                  <inactive>
                   <colorrole role="WindowText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>173</red>
                      <green>173</green>
                      <blue>173</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </inactive>
                  <disabled>
                   <colorrole role="WindowText">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>120</red>
                      <green>120</green>
                      <blue>120</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </disabled>
                 </palette>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"> </string>
                </property>
                <property name="text">
                 <string>unknown</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="topMargin">
               <number>14</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="island_access_label">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>Access:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="island_access_address">
                <property name="font">
                 <font>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string notr="true"/>
                </property>
                <property name="openExternalLinks">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="island_admin_access_label">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>Admin Access: </string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="island_admin_access_address">
                <property name="font">
                 <font>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string notr="true"/>
                </property>
                <property name="openExternalLinks">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="visible">
               <bool>true</bool>
              </property>
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="setup_required_reason">
                   <property name="text">
                    <string>Island VM not found</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="setup_required_icon">
                   <property name="maximumSize">
                    <size>
                     <width>16</width>
                     <height>16</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="pixmap">
                    <pixmap resource="resources.qrc">:/images/info</pixmap>
                   </property>
                   <property name="scaledContents">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="button_launch_setup">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::RightToLeft</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>Run setup</string>
                 </property>
                 <property name="icon">
                  <iconset resource="resources.qrc">
                   <normaloff>:/images/gear</normaloff>:/images/gear</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="launchIslandButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="whatsThis">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Launch island. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Launch Island</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/play</normaloff>:/images/play</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="launchMode">
              <item>
               <property name="text">
                <string>Quiet</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Normal</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QPushButton" name="shutdownIslandButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Shutdown Island</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/stop</normaloff>:/images/stop</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>48</width>
                <height>48</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="stopMode">
              <item>
               <property name="text">
                <string>Soft</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Force</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="restartIslandButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 20px;</string>
            </property>
            <property name="text">
             <string>Restart Island</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/images/reload</normaloff>:/images/reload</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabIslandSettings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="vboxmanagePathLabel">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Path to vboxmanage:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="vboxmanagePathLineEdit">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="vMnameLabel">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Island VM name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="vmnameLineEdit">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>125</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Data folder path:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>774</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionInfo"/>
    <addaction name="separator"/>
    <addaction name="actionMinimize_2"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="text">
    <string>Minimize</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionMinimize_2">
   <property name="text">
    <string>Minimize</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
